data/human/news.txt

```diff
@@ -1467,10 +1609,27 @@ news "human republic reporter in pirate space"
 			"Look at this! 'Top Ten Pirates Hector Barbarossa is Too Afraid to Cross!'"
 			"Read this! 'Twenty Pirates with the Highest IQs - You'll Never Guess Who's Number One!'"
 			"Check this out! 'Telltale Signs Your Spouse is Secretly a Pirate - Number Five Says it All!'"
+			"Read my column on how to identify pirates in disguise!... Or just look around this planet, that also works."
 		word
 			`"`
 
 
+
+news "spaceport video advertisement"
+	location
+		government "Republic"
+	name
+		word
+			"Spaceport Video Screen"
+	message
+		word
+			"You see a video of stuntmen making a jump from the edge of space. The 'do not try this at home', 'illegal on many planets', and 'high risk/deadly' warnings at the end of the video suggest that this might become quite a popular sport."
```

Comment:
```suggestion
			`You see a video of stuntmen making a jump from the edge of space. The "do not try this at home", "illegal on many planets", and "high risk/deadly" warnings at the end of the video suggest that this might become quite a popular sport.`
```